packages/opentelemetry-core/src/trace/TraceState.ts

```diff
@@ -68,10 +68,11 @@ export class TraceState implements api.TraceState {
       .split(LIST_MEMBERS_SEPARATOR)
       .reverse() // Store in reverse so new keys (.set(...)) will be placed at the beginning
       .reduce((agg: Map<string, string>, part: string) => {
-        const i = part.indexOf(LIST_MEMBER_KEY_VALUE_SPLITTER);
+        const listMember = part.trim(); // Optional Whitespace (OWS) handling
```

Comment:
This adds the OWS support / fixes those related errors. While it does add an additional string, there's likely other more impactful optimizations we could do in this area if it became a concern.